drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -2374,6 +2374,16 @@ matterGeneric:
     deviceTypes:
       - id: 0x010B # Dimmable Plug-in Unit
     deviceProfileName: plug-level
+  - id: "matter/electrical-sensor"
+    deviceLabel: Matter Electrical Sensor
+    deviceTypes:
+      - id: 0x0510 # Electrical Sensor
```

Comment:
If this is using the `plug-* profile shouldn't it also have the on/off plugin unit as a required device type?

Reply:
Yes, I think that is a good idea. If we do that though, should we also change the device label and id to something that indicates this is a plug-specific generic fingerprint?

Reply:
Yeah, I'd probably just go with "Matter OnOff Plug" like the regular plug

Reply:
So we won't really have "just" an electrical sensor generic fingerprint?

Reply:
Actually, scratch my question, that makes sense to me.